Chart: Update axis grid topics (#7650)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChart/1 Configuration/commonAxisSettings/minorGrid/minorGrid.md

                color="blue"
                [opacity]="0.1"
                [width]="1"
            ></dxo-minor-grid>
        </dxo-common-axis-settings>
    </dx-chart>

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxChart ... >
            <DxCommonAxisSettings>
                <DxMinorGrid
                    :visible="true"
                    color="blue"
                    :opacity="0.1"
                    :width="1"
                />
            </DxCommonAxisSettings>
        </DxChart>
    </template>

    <script setup lang="ts">
    import { DxChart, DxCommonAxisSettings, DxMinorGrid } from 'devextreme-vue/chart';

    </script>

##### React

    <!-- tab: App.tsx -->
    import { Chart, CommonAxisSettings, MinorGrid } from 'devextreme-react/chart';

    function App() {
        return (
            <Chart ... >
                <CommonAxisSettings>
                    <MinorGrid
                        visible={true}
                        color="blue"
                        opacity={0.1}
                        width={1}
                    />
                </CommonAxisSettings>
            </Chart>
        )
    }

---

> The **commonAxisSettings**.[minorGrid](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/commonAxisSettings/minorGrid '/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/minorGrid/') object specifies common settings for all minor grid lines in the chart. To configure only those grid lines that descend from a particular axis, use the following objects.

Reviewer: ```suggestion
The **commonAxisSettings**.[minorGrid](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/commonAxisSettings/minorGrid '/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/minorGrid/') object specifies common settings for all minor gridlines in a chart. To configure only those gridlines that descend from an axis of your choice, use the following objects.
```

Axis-specific settings override common settings.

[note]

> - Displaying the minor grid alone may impair chart readability. We recommend you implement the minor grid in conjunction with the major grid.

Reviewer: ```suggestion
- If you display the minor grid alone, it may impair chart readability. We recommend you implement the minor grid in conjunction with the major grid.
```
